### DISCUSSION TITLE
Cannot accept an invitation to repo

### DISCUSSION THREAD

**Discussion by abeer-work** [community/community#29167] [Other Feature Feedback, Questions, & Ideas] [answered]
I was sent an invitation to a private repo, and every time I try to accept the invitation on the browser, I end up on a page with an infinite scroll where the invitation is at the bottom, never being able to actually click on it. If I try to click on it, it the scrollbar increases and the "Accept invitation" panel moves further to the bottom. It's like Mario 64's infinite staircase kinda issue.

**Comment by augustoaraujoo** [comment]
check the invite in your email

**Comment by abeer-work** [comment]
This is what happens when I click on the link from the email. If I try to click "Accept Invite," currently at the bottom of this page, I get an infinite scroll: 

<img width="1390" alt="Screen Shot 2022-08-11 at 3 28 24 PM" src="https://user-images.githubusercontent.com/102306515/184223689-1be64b4c-7a2a-4570-a75c-947d30ee4075.png">

**Comment by radar-blip** [comment, marked as answer]
Someone else is having this problem also. Here: #29187
Seems to be a glitch.
You can file a bug report here:

https://support.github.com/contact/bug-report

Sooner the better!

**Reply by abeer-work** [reply]
Thank you will do!

**Reply by radar-blip** [reply]
You're welcome. Hope all goes well.

**Comment by totonagano** [comment]
Has a workaround ever been found for this? I am experiencing the same thing...

**Comment by DefaultO** [comment]
@totonagano pretty late, but I had the same problem that it would link me to the private repositories invitation page: ``https://github.com/[github username]/[repository name]/invitations`` instead of your personal settings organization page: https://github.com/settings/organizations. Hope this helps anybody new to GitHub getting invited to a private repository, as the issue still remains, that GitHub will lead you to the repository that you got no access to, to accept the invite, instead into your personal settings -> organizations.
